lib/wallet/payment_history_list_item.dart

```diff
@@ -58,19 +64,42 @@ class PaymentHistoryListItem extends StatelessWidget {
 
     final amountDisplay = data.amount.display(sign: true, currency: Currency.sat);
 
-    return ListTile(
-      leading: statusIcon,
-      title: Text(data.type.display),
-      trailing: Row(
-        mainAxisSize: MainAxisSize.min,
-        mainAxisAlignment: MainAxisAlignment.center,
-        crossAxisAlignment: CrossAxisAlignment.baseline,
-        textBaseline: TextBaseline.alphabetic,
-        children: [
-          AmountItem(text: amountDisplay.value, unit: AmountUnit.satoshi, iconColor: Colors.grey)
-        ],
-      ),
-    );
+    final date = data.status == PaymentStatus.pending
+        ? "pending"
+        : DateFormat.yMMMd()
```

Comment:
can we add a bit more precision (hours and minutes?). I would like to see that in my payment history.

Reply:
Done: 
<img width="425" alt="image" src="https://user-images.githubusercontent.com/224613/203222700-612edc74-ac82-4c1f-83b9-3e87c645d468.png">